tools/erc-repository-indexer/erc-contract-indexer/src/utils/constants.ts

```diff
@@ -0,0 +1,25 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+export default {
+  RETRY_DELAY_MS: 15000,
+  GET_CONTRACT_ENDPOINT: '/api/v1/contracts',
+  MIRROR_NODE_FALL_BACK_BASE_URL: 'https://previewnet.mirrornode.hedera.com',
```

Comment:
Should we hardcode previewnet here?

Reply:
Are you referring to the `HEDERA_NETWORK`, or are you discussing the `MIRROR_NODE_FALL_BACK_BASE_URL` being set to previewnet? 

If it's about the `HEDERA_NETWORK`, it may not be necessary and could potentially be removed from the .env file., 

If you're referring to the `MIRROR_NODE_FALL_BACK_BASE_URL` being set to previewnet by default, it's simply a safe fallback, ensuring that the script has a valid URL to revert to.